modules/hostdb/hostweight_test.go

```diff
@@ -0,0 +1,54 @@
+package hostdb
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+func calculateWeightFromUInt64Price(price uint64) (weight types.Currency) {
+	entry := hostEntry{
+		HostSettings: modules.HostSettings{
+			Price: types.NewCurrency64(price),
+		},
+	}
```

Comment:
since `HostSettings` is embedded, you can access it a little more compactly with:
```go
var entry hostEntry
entry.Price = types.NewCurrency64(price)
```